alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/DefaultConnectionInfoRepository.java

```diff
@@ -62,15 +53,15 @@ class DefaultConnectionInfoRepository implements ConnectionInfoRepository {
   @Override
   public ConnectionInfo getConnectionInfo(InstanceName instanceName, KeyPair keyPair)
       throws ExecutionException, InterruptedException, CertificateException, ApiException {
-    Future<com.google.cloud.alloydb.v1beta.ConnectionInfo> infoFuture =
+    Future<com.google.cloud.alloydb.v1.ConnectionInfo> infoFuture =
         executor.submit(() -> getConnectionInfo(instanceName));
     Future<GenerateClientCertificateResponse> clientCertificateResponseFuture =
         executor.submit(() -> getGenerateClientCertificateResponse(instanceName, keyPair));
 
-    com.google.cloud.alloydb.v1beta.ConnectionInfo info = infoFuture.get();
+    com.google.cloud.alloydb.v1.ConnectionInfo info = infoFuture.get();
 
     GenerateClientCertificateResponse certificateResponse = clientCertificateResponseFuture.get();
-    ByteString pemCertificateBytes = certificateResponse.getPemCertificateBytes();
+    ByteString pemCertificateBytes = certificateResponse.getPemCertificateChainBytes(0);
     X509Certificate clientCertificate = parseCertificate(pemCertificateBytes);
```

Comment:
As you pointed out, this is the problem. We need to remove client certificate entirely here and elsewhere. Instead we'll use the PEM certificate chain (which includes the client cert at index 0).

Reply:
Committed the changes. The integration tests are still failing due to the issues in the `v1` AlloyDB API. 

We should be able to merge this PR in a week when the fix gets to `prod`.